src/components/Home.vue

```diff
@@ -23,16 +25,42 @@ export default {
       todos: []
     };
   },
+  mounted() {
+    todoService.getTodos().then(data => {
+      this.todos = [...data];
+    });
+  },
   methods: {
     addTodo(newTodoObj) {
-      this.todos = [...this.todos, newTodoObj];
```

Comment:
method is handling couple of things and name is not telling us that. Maybe to divide logic for checking is user authorized and sending request. It would be good to add error handling and to display api errors to user.